File: VB/EntityFrameworkCodeFirstBindingExample/CarScheduling.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace EntityFrameworkCodeFirstBindingExample

    <Table("CarScheduling")>
    Public Partial Class CarScheduling

        Public Property ID As Integer

        Public Property CarId As Integer?

        Public Property UserId As Integer?

        Public Property Status As Integer?

        <StringLength(50)>
        Public Property Subject As String

        Public Property Description As String

        Public Property Label As Integer?

        Public Property StartTime As Date?

        Public Property EndTime As Date?

        <StringLength(50)>
        Public Property Location As String

        Public Property AllDay As Boolean

        Public Property EventType As Integer?

        Public Property RecurrenceInfo As String

        Public Property ReminderInfo As String

        <Column(TypeName:="smallmoney")>
        Public Property Price As Decimal?

        Public Property ContactInfo As String
    End Class
End Namespace

File: CS/EntityFrameworkCodeFirstBindingExample/CarScheduling.cs
namespace EntityFrameworkCodeFirstBindingExample
{
    using System;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    [Table("CarScheduling")]
    public partial class CarScheduling
    {
        public int ID { get; set; }

        public int? CarId { get; set; }

        public int? UserId { get; set; }

        public int? Status { get; set; }

        [StringLength(50)]
        public string Subject { get; set; }

        public string Description { get; set; }

        public int? Label { get; set; }

        public DateTime? StartTime { get; set; }

        public DateTime? EndTime { get; set; }

        [StringLength(50)]
        public string Location { get; set; }

        public bool AllDay { get; set; }

        public int? EventType { get; set; }

        public string RecurrenceInfo { get; set; }

        public string ReminderInfo { get; set; }

        [Column(TypeName = "smallmoney")]
        public decimal? Price { get; set; }

        public string ContactInfo { get; set; }
    }
}
